### DISCUSSION TITLE
Why are there two different github.com token environment variables?

### DISCUSSION THREAD

**Discussion by HenrikPoulsen** [renovatebot/renovate#17627] [Request Help] [answered]
I noticed that there is now `GITHUB_TOKEN` which sets up a host rule for api.github.com.
Why does it simply not just use `GITHUB_COM_TOKEN` for this (which sets up a similar host rule), since it confuses the situation for github enterprise.
I would imagine `GITHUB_TOKEN` would be the enterprise instance when hosted on github enterprise but that is not the case.

**Comment by rarkins** [comment, marked as answer]
Where is this?

**Reply by HenrikPoulsen** [reply]
I just noticed that the `GITHUB_TOKEN` is in `tools/docs/manager.ts` which isn't relevant for running renovate.
So I will close this. Sorry for the noise
